.github/labeler.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Labeler
+
+libraries:
```

Comment:
We don't have this label, no? We do have C, C++ and Python which are used below.

Reply:
I believe my initial intention with `libraries` and `module`, was to be able to highlight evtl. changes to core vs. modules. Changes to libraries, being base to all, perhaps need particular attention.

Reply:
I think you need to create the labels first. I think they might be useful, so why not, especially if we have automation to help us label.